build.gradle

```diff
@@ -7,7 +7,7 @@ plugins {
 sourceCompatibility = JavaVersion.VERSION_17
 targetCompatibility = sourceCompatibility
 // Unknown JVM target version: 17
-compileKotlin.kotlinOptions.jvmTarget = "16"//sourceCompatibility.toString()
```

Comment:
Is this issue solved by the Kotlin version bump below?

Reply:
I actually don't know, but I suppose the new Kotlin version would have Java 17 support.
But don't know, what would happen otherwise, if it didn't work

Reply:
It would throw an error during compilation. If J17 is supported now then I'll accept the bumped kotlin version